packages/core/src/services/zoom.ts

```diff
@@ -64,4 +64,22 @@ export class Zoom extends Service {
 			extent(zoomBarData, (d: any) => d[domainIdentifier])
 		);
 	}
+
+	isTopZoomBarLoading() {
```

Comment:
This should be `isZoomBarLoading(position)`

Reply:
Updated in https://github.com/carbon-design-system/carbon-charts/pull/776/commits/9436b0d67a17883273146053a5a6d799dac6c494.